docs/user_guide/discretisation/IncreasingWidthDiscretiser.rst

```diff
@@ -0,0 +1,134 @@
+.. _increasing_width_discretiser:
+
+.. currentmodule:: feature_engine.discretisation
+
+IncreasingWidthDiscretiser
+==========================
+
+The :class:`IncreasingWidthDiscretiser()` divides continuous numerical variables into
+intervals of increasing width with equal increments. Note that the
+proportion of observations per interval may vary.
```

Comment:
An example of the interval calculation would be extremely helpful, like the table the person uses in their slides. Maybe with different numbers, or alternatively clearly stating where the table was taken from.